server/fleet/hosts.go

```diff
@@ -333,7 +333,9 @@ func PlatformFromHost(hostPlatform string) string {
 	switch {
 	case IsLinux(hostPlatform):
 		return "linux"
-	case hostPlatform == "darwin", hostPlatform == "windows":
+	case hostPlatform == "darwin", hostPlatform == "windows",
+		// Some customers have custom agents that support ChromeOS
+		hostPlatform == "CrOS":
 		return hostPlatform
```

Comment:
Double checking the reasoning: 

We use `PlatformFromHost()/FleetPlatform()` to know which set of policies to send to a device. Fleet currently support targeting policies for the three main OSs, `"linux"`, `"darwin"` and `"windows"`.

This change would make it so that these Chrome OS devices wouldn't receive any policies to run.

Guessing we want to do this because osquery tables haven't been tested on Chrome OSs; also because generic Linux policy queries may not run well on these devices.

Reply:
I think Policies are already not running for these ChromeOS devices because of this empty platform. Mostly the intent of this is to avoid all of the error logs like https://github.com/fleetdm/fleet/blob/05426bcdca287b6a3485044720a378ff6e2efab1/server/datastore/mysql/hosts.go#L1618.

The UI isn't going to support policies, but I think with this change they could now be configured by `fleetctl` yaml?

Reply:
Ah gotcha. Makes sense.

> The UI isn't going to support policies, but I think with this change they could now be configured by fleetctl yaml?

Who knows, only one way to find out :) (my guess is that we have a check somewhere that it must be one of the three main OSs, but again, we should double-check/test-it).

My gut feeling says we should test this change somehow (with a modified osquery?).
But we could merge+release today and in the catastrophic scenario that one of us find some issue on Monday, then ship a point release.

Reply:
Where/how do you think things might break with this change? I would like to dig into this deeper given your gut feeling.

Reply:
Upon second look I think we are a-ok. 

Let's just check that the MySQL SQL queries that will now run with a value other than `"linux"`, `"windows"` and `"darwin"` (`"CrOS"`) continue to work (do not break).
E.g. https://github.com/fleetdm/fleet/blob/3016ac11364d77ef5d627c5a477478c5c1cd9f7a/server/datastore/mysql/policies.go#L377